### PR TITLE
Branding stuff

### PR DESCRIPTION
**Pull request by isislovecruft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 
 # curve25519-dalek  [![](https://img.shields.io/crates/v/curve25519-dalek.svg)](https://crates.io/crates/curve25519-dalek) [![](https://docs.rs/curve25519-dalek/badge.svg)](https://docs.rs/curve25519-dalek) [![](https://travis-ci.org/isislovecruft/curve25519-dalek.svg?branch=master)](https://travis-ci.org/isislovecruft/curve25519-dalek)
 
-**A low-level cryptographic library for point, group, field, and scalar
+<table>
```
Comment (tarcieri) [inline README.md]: Not sure if you like the border or not, but you might consider `border="0"` (omg old skool)

**Comment by tarcieri** [comment]
Hmm, maybe try `` instead of a table?

https://stackoverflow.com/questions/19075023/flow-text-around-an-image-in-github-markdown

**Comment by isislovecruft** [comment]
lol, thanks for looking out for my shitty webdev attempts, I love you @tarcieri

**Comment by isislovecruft** [comment]
Aha, that seems to have worked, thanks!

**Comment by tarcieri** [comment]
Woohoo, looks good

**Pull request merged**